includes/admin/class-upgrades.php

```
@@ -452,7 +452,19 @@ private function v195_upgrade() {
 	private function v20_upgrade() {
 		@affiliate_wp()->capabilities->add_caps();
 		$this->log( 'Upgrade: Core capabilities have been upgraded.' );
+
+
+		// Update settings
+		affiliate_wp()->settings->set( array(
```

Let's silence any potential errors here with the `@` for consistency with the other upgrade routines.